src/irmin-pack/unix/errors.ml

```diff
@@ -38,7 +38,7 @@ type base_error =
   [ `Double_close
   | `File_exists of string
   | `Invalid_parent_directory
-  | `No_such_file_or_directory
+  | `No_such_file_or_directory of string
```

Comment:
This is great, thank you!

Reply:
I'd like to take a look at all of our errors to decide which ones really deserve some extra context info. But, for a future PR. :smile: